sqlmesh/schedulers/airflow/integration.py

```diff
@@ -123,29 +124,27 @@ def dags(self) -> t.List[DAG]:
         """
         self.set_default_catalog(self._default_catalog)
         with util.scoped_state_sync() as state_sync:
-            stored_snapshots = state_sync.get_snapshots(None)
```

Comment:
so this is the main change right? you got rid of this

Reply:
exactly